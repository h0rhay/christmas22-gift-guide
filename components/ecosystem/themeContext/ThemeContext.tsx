import React, { ReactChild, ReactNode } from "react";
import styled, { ThemeProvider } from "styled-components";
import theme from "styles/theme";

const ColoredSurface = styled.div`
  background-color: ${({ theme }) => theme.currentColorGroup.surface};
  * {
    color: ${({ theme }) => theme.currentColorGroup.interface};
  }
`;

type ThemeContextProps = {
  colorGroup?: string;
  children: ReactChild | Array<ReactChild> | ReactNode;
};

const ThemeContext = ({
  children,
  colorGroup = theme.fallbackColorGroup,
}: ThemeContextProps) => {
  const currentColorGroup =
    theme.colorsGroup[colorGroup] ||
    theme.colorsGroup[theme.fallbackColorGroup];
  return (
    <ThemeProvider theme={{ ...theme, currentColorGroup }}>
      <ColoredSurface className="colored-surface">{children}</ColoredSurface>
    </ThemeProvider>
  );
};

export default ThemeContext;
