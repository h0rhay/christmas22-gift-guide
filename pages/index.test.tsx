import { render } from "@testing-library/react";
import LandingPage from "./index.page";
import { ThemeProvider } from "styled-components";
import Theme from "__mocks__/mockTheme";
import ThemeContext from "components/ecosystem/themeContext/ThemeContext";

const mockLandingPageData = {
  desc: "Dolore aliquip nostrud culpa ut nisi do do magna elit. Proident quis nostrud adipisicing ea aliqua incididunt. Lorem occaecat quis esse velit exercitation fugiat ipsum reprehenderit nulla ex ut deserunt. Officia dolore eiusmod et nulla aliquip ea ad cillum.",
  pullQuote: "I am a pullquote",
  landingPageImages: [
    {
      src: "/img/homePage/xmas-glitter-ball-reclining-lady-rct.webp",
      alt: "Lady reclining under giant glitter ball in the style of a christmas pudding",
    },
    {
      src: "/img/homePage/xmas-glitter-ball-reclining-lady-rct-2.webp",
      alt: "Lady reclining under giant glitter ball in the style of a christmas pudding",
    },
    {
      src: "/img/homePage/xmas-glitter-ball-reclining-lady-rct-3.webp",
      alt: "Lady reclining under giant glitter ball in the style of a christmas pudding",
    },
  ]
};
const renderComponent = () =>
  render(
    <ThemeProvider theme={Theme}>
      <ThemeContext colorGroup="hero">
        <LandingPage
          randomIconNamePosition={0}
          landingPageData={mockLandingPageData}
          categoryList={[]}
          children={undefined}
        />
      </ThemeContext>
    </ThemeProvider>
  );

describe("Welcome page", () => {
  beforeEach(() => {
    jest.spyOn(global.Math, "floor").mockReturnValue(1);
  });

  afterEach(() => {
    jest.spyOn(global.Math, "floor").mockRestore();
  });
  it("should match snapshot", () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
