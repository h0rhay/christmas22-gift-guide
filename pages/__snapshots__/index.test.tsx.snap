// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome page should match snapshot 1`] = `
<div>
  <div
    class="ThemeContext__ColoredSurface-sc-7o1m9v-0 cumPXi colored-surface"
  >
    <div
      class="indexpage__PageWrapper-sc-4zumhw-0 cfpvpp"
    >
      <div
        class="ThemeContext__ColoredSurface-sc-7o1m9v-0 cumPXi colored-surface"
      >
        <section
          class="siteWidthWrapper__SiteWidthContainer-sc-1kx0urc-0 ckJkzr homePageHero__StyledHomepageHero-sc-174cec1-0 jkBTdd"
        >
          <div>
            <h1>
              <div
                class="xmasTitleMobile__StyledAnimationWrap-sc-1csolml-0 krBsHZ"
              >
                <div>
                  <svg
                    height="250"
                    preserveAspectRatio="xMidYMid meet"
                    style="width: 100%; height: 100%; transform: translate3d(0,0,0);"
                    viewBox="0 0 1080 250"
                    width="1080"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <defs>
                      <clippath
                        id="__lottie_element_2"
                      >
                        <rect
                          height="250"
                          width="1080"
                          x="0"
                          y="0"
                        />
                      </clippath>
                    </defs>
                    <g
                      clip-path="url(#__lottie_element_2)"
                    >
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        opacity="1"
                        style="display: block;"
                        transform="matrix(1.100000023841858,0,0,1.1000038385391235,-1028.803466796875,-198.10089111328125)"
                      >
                        <g
                          opacity="1"
                          transform="matrix(1,0,0,1,876.4940185546875,292.9530029296875)"
                        >
                          <path
                            d=" M52.0890007019043,50.98500061035156 C52.0890007019043,90.03900146484375 37.31999969482422,111.19300079345703 1.0740000009536743,111.19300079345703 C-34.369998931884766,111.19300079345703 -51.55400085449219,96.5479965209961 -51.55400085449219,32.54399871826172 C-51.55400085449219,32.54399871826172 -6.714000225067139,32.54399871826172 -6.714000225067139,32.54399871826172 C-6.714000225067139,61.5629997253418 -3.759999990463257,65.9010009765625 1.0740000009536743,65.9010009765625 C6.980999946594238,65.9010009765625 7.249000072479248,60.74800109863281 7.249000072479248,50.98500061035156 C7.249000072479248,37.96799850463867 6.175000190734863,32.27299880981445 -10.741000175476074,17.89900016784668 C-30.340999603271484,1.6269999742507935 -52.09000015258789,-10.57699966430664 -52.09000015258789,-53.426998138427734 C-52.09000015258789,-90.30999755859375 -36.78499984741211,-111.19300079345703 -1.88100004196167,-111.19300079345703 C32.75699996948242,-111.19300079345703 48.86800003051758,-90.30999755859375 48.86800003051758,-43.391998291015625 C48.86800003051758,-43.391998291015625 4.296000003814697,-43.391998291015625 4.296000003814697,-43.391998291015625 C4.296000003814697,-61.020999908447266 2.4159998893737793,-65.9020004272461 -1.88100004196167,-65.9020004272461 C-6.443999767303467,-65.9020004272461 -7.25,-61.020999908447266 -7.25,-53.426998138427734 C-7.25,-39.053001403808594 -1.0750000476837158,-33.900001525878906 13.694000244140625,-22.510000228881836 C36.24700164794922,-5.1529998779296875 52.0890007019043,9.762999534606934 52.0890007019043,50.98500061035156z"
                            fill="rgb(218,64,49)"
                            fill-opacity="1"
                          />
                          <path
                            d=" M52.0890007019043,50.98500061035156 C52.0890007019043,90.03900146484375 37.31999969482422,111.19300079345703 1.0740000009536743,111.19300079345703 C-34.369998931884766,111.19300079345703 -51.55400085449219,96.5479965209961 -51.55400085449219,32.54399871826172 C-51.55400085449219,32.54399871826172 -6.714000225067139,32.54399871826172 -6.714000225067139,32.54399871826172 C-6.714000225067139,61.5629997253418 -3.759999990463257,65.9010009765625 1.0740000009536743,65.9010009765625 C6.980999946594238,65.9010009765625 7.249000072479248,60.74800109863281 7.249000072479248,50.98500061035156 C7.249000072479248,37.96799850463867 6.175000190734863,32.27299880981445 -10.741000175476074,17.89900016784668 C-30.340999603271484,1.6269999742507935 -52.09000015258789,-10.57699966430664 -52.09000015258789,-53.426998138427734 C-52.09000015258789,-90.30999755859375 -36.78499984741211,-111.19300079345703 -1.88100004196167,-111.19300079345703 C32.75699996948242,-111.19300079345703 48.86800003051758,-90.30999755859375 48.86800003051758,-43.391998291015625 C48.86800003051758,-43.391998291015625 4.296000003814697,-43.391998291015625 4.296000003814697,-43.391998291015625 C4.296000003814697,-61.020999908447266 2.4159998893737793,-65.9020004272461 -1.88100004196167,-65.9020004272461 C-6.443999767303467,-65.9020004272461 -7.25,-61.020999908447266 -7.25,-53.426998138427734 C-7.25,-39.053001403808594 -1.0750000476837158,-33.900001525878906 13.694000244140625,-22.510000228881836 C36.24700164794922,-5.1529998779296875 52.0890007019043,9.762999534606934 52.0890007019043,50.98500061035156z"
                            fill-opacity="0"
                            stroke="rgb(255,254,255)"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                            stroke-opacity="1"
                            stroke-width="2"
                          />
                        </g>
                      </g>
                    </g>
                  </svg>
                </div>
              </div>
              <span
                class="sr-only"
              >
                Seasons
              </span>
            </h1>
            <div
              class="imageFrame__Container-sc-180q2f3-0 cA-DGOH imageframe__container mobile-image-frame"
            >
              <div
                class="imageframe__image-container"
              >
                <div
                  class="imageComponent__StyledImage-sc-y6629v-0 iprdOX image-component__container"
                  type=""
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      alt="Lady reclining under giant glitter ball in the style of a christmas pudding"
                      class="imageframe__image"
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; filter: blur(20px); background-size: cover; background-image: url(/img/homePage/xmas-glitter-ball-reclining-lady-rct-2.webp); background-position: 0% 0%;"
                    />
                    <noscript />
                  </div>
                </div>
              </div>
              <div
                class="frame__Container-sc-he37x9-0 gFwSUZ svg-frame-container svg__frame--mobile"
              >
                <svg
                  class="svg__frame--mobile"
                  viewBox="0 0 1180 1450"
                  xml:space="preserve"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="frame__corner"
                    d="M0 0v1450h1180V0H0zm1113.7 1222.6c0 9.8-7.3 18.1-17.1 19.6-62.8 9.5-112.5 59.9-121.9 123.4-1.4 9.9-9.8 17.2-19.7 17.2H225c-9.9 0-18.2-7.4-19.7-17.2-9.3-63.5-59.1-113.8-121.9-123.4-9.8-1.5-17.1-9.7-17.1-19.6V225.4c0-9.8 7.3-18.1 17.1-19.6 62.8-9.5 112.5-59.9 121.9-123.4 1.4-9.9 9.8-17.2 19.7-17.2h730c10 0 18.3 7.3 19.7 17.2 9.3 63.5 59.1 113.8 121.9 123.4 9.8 1.5 17.1 9.7 17.1 19.6v997.2z"
                  />
                  <path
                    class="frame__in"
                    d="M966.3 50H213.7c-10.2 0-18.8 7.6-20.3 17.7-9.7 65.4-61 117.4-125.7 127.3-10.2 1.5-17.7 10-17.7 20.2v1019.6c0 10.2 7.5 18.7 17.7 20.2 64.7 9.9 116 61.8 125.7 127.3 1.5 10.1 10.1 17.7 20.3 17.7h752.7c10.2 0 18.8-7.6 20.3-17.7 9.7-65.5 61-117.4 125.7-127.3 10.1-1.5 17.7-10.1 17.7-20.2V215.2c0-10.2-7.5-18.7-17.7-20.2-64.7-9.9-116-61.8-125.7-127.3-1.5-10.1-10.1-17.7-20.4-17.7z"
                  />
                  <path
                    class="frame__out"
                    d="M955 66.2H225c-9.9 0-18.3 7.3-19.7 17.2-9.4 63.5-59.1 113.9-121.9 123.4-9.8 1.5-17.1 9.8-17.1 19.6v997.1c0 9.9 7.3 18.1 17.1 19.6 62.8 9.6 112.6 59.9 121.9 123.4 1.5 9.8 9.8 17.2 19.7 17.2h730c9.9 0 18.3-7.3 19.7-17.2 9.4-63.5 59.1-113.9 121.9-123.4 9.8-1.5 17.1-9.8 17.1-19.6V226.4c0-9.9-7.3-18.1-17.1-19.6-62.8-9.6-112.6-59.9-121.9-123.4-1.4-9.9-9.7-17.2-19.7-17.2z"
                  />
                </svg>
              </div>
              <div
                class="frame__Container-sc-he37x9-0 dXCvsx svg-frame-container svg__frame--desktop"
              >
                <svg
                  class="svg__frame--desktop"
                  viewBox="0 0 2020 1580"
                  xml:space="preserve"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="frame__corner"
                    d="M0,0v1580h2020V0H0z M1953.7,1155c0,9.9-7.4,18.2-17.2,19.7c-63.5,9.3-113.8,59.1-123.4,121.9
    c-1.5,9.8-9.7,17.1-19.6,17.1H226.4c-9.8,0-18.1-7.3-19.6-17.1c-9.5-62.8-59.9-112.5-123.4-121.9c-9.9-1.4-17.2-9.8-17.2-19.7V425
    c0-10,7.3-18.3,17.2-19.7c63.5-9.3,113.8-59.1,123.4-121.9c1.5-9.8,9.7-17.1,19.6-17.1h1567.1c9.8,0,18.1,7.3,19.6,17.1
    c9.5,62.8,59.9,112.5,123.4,121.9c9.9,1.4,17.2,9.8,17.2,19.7V1155z"
                  />
                  <path
                    class="frame__in"
                    d="M50,413.7v752.7c0,10.2,7.6,18.8,17.7,20.3c65.4,9.7,117.4,61,127.3,125.7c1.5,10.1,10.1,17.7,20.2,17.7
		h1589.6c10.2,0,18.7-7.5,20.2-17.7c9.9-64.7,61.8-116,127.3-125.7c10.1-1.5,17.7-10.1,17.7-20.3V413.7c0-10.2-7.6-18.8-17.7-20.3
		c-65.5-9.7-117.4-61-127.3-125.7c-1.5-10.2-10-17.7-20.2-17.7H215.2c-10.2,0-18.7,7.5-20.2,17.7c-9.9,64.7-61.8,116-127.3,125.7
		C57.6,394.8,50,403.4,50,413.7z"
                  />
                  <path
                    class="frame__out"
                    d="M66.2,425v730c0,9.9,7.3,18.3,17.2,19.7c63.5,9.4,113.9,59.1,123.4,121.9c1.5,9.8,9.8,17.1,19.6,17.1h1567.1
		c9.9,0,18.1-7.3,19.6-17.1c9.6-62.8,59.9-112.6,123.4-121.9c9.8-1.5,17.2-9.8,17.2-19.7V425c0-9.9-7.3-18.3-17.2-19.7
		c-63.5-9.4-113.9-59.1-123.4-121.9c-1.5-9.8-9.8-17.1-19.6-17.1H226.4c-9.9,0-18.1,7.3-19.6,17.1c-9.6,62.8-59.9,112.6-123.4,121.9
		C73.5,406.7,66.2,415,66.2,425z"
                  />
                </svg>
              </div>
            </div>
            <h1>
              <div
                class="xmasTitleMobile__StyledAnimationWrap-sc-1csolml-0 krBsHZ"
              >
                <div>
                  <svg
                    height="250"
                    preserveAspectRatio="xMidYMid meet"
                    style="width: 100%; height: 100%; transform: translate3d(0,0,0);"
                    viewBox="0 0 1080 250"
                    width="1080"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <defs>
                      <clippath
                        id="__lottie_element_29"
                      >
                        <rect
                          height="250"
                          width="1080"
                          x="0"
                          y="0"
                        />
                      </clippath>
                    </defs>
                    <g
                      clip-path="url(#__lottie_element_29)"
                    >
                      <g
                        opacity="1"
                        style="display: block;"
                        transform="matrix(1.100000023841858,0,0,1.1000038385391235,857.1972045898438,-480.5494689941406)"
                      >
                        <g
                          opacity="1"
                          transform="matrix(1,0,0,1,247.697998046875,550.0430297851562)"
                        >
                          <path
                            d=" M2.0139999389648438,-67.1259994506836 C2.0139999389648438,-67.1259994506836 2.0139999389648438,-9.397000312805176 2.0139999389648438,-9.397000312805176 C2.0139999389648438,-9.397000312805176 38.262001037597656,-9.397000312805176 38.262001037597656,-9.397000312805176 C38.262001037597656,-9.397000312805176 38.262001037597656,30.878000259399414 38.262001037597656,30.878000259399414 C38.262001037597656,30.878000259399414 2.0139999389648438,30.878000259399414 2.0139999389648438,30.878000259399414 C2.0139999389648438,30.878000259399414 2.0139999389648438,107.4010009765625 2.0139999389648438,107.4010009765625 C2.0139999389648438,107.4010009765625 -38.262001037597656,107.4010009765625 -38.262001037597656,107.4010009765625 C-38.262001037597656,107.4010009765625 -38.262001037597656,-107.4010009765625 -38.262001037597656,-107.4010009765625 C-38.262001037597656,-107.4010009765625 38.262001037597656,-107.4010009765625 38.262001037597656,-107.4010009765625 C38.262001037597656,-107.4010009765625 38.262001037597656,-67.1259994506836 38.262001037597656,-67.1259994506836 C38.262001037597656,-67.1259994506836 2.0139999389648438,-67.1259994506836 2.0139999389648438,-67.1259994506836z"
                            fill="rgb(218,64,49)"
                            fill-opacity="1"
                          />
                          <path
                            d=" M2.0139999389648438,-67.1259994506836 C2.0139999389648438,-67.1259994506836 2.0139999389648438,-9.397000312805176 2.0139999389648438,-9.397000312805176 C2.0139999389648438,-9.397000312805176 38.262001037597656,-9.397000312805176 38.262001037597656,-9.397000312805176 C38.262001037597656,-9.397000312805176 38.262001037597656,30.878000259399414 38.262001037597656,30.878000259399414 C38.262001037597656,30.878000259399414 2.0139999389648438,30.878000259399414 2.0139999389648438,30.878000259399414 C2.0139999389648438,30.878000259399414 2.0139999389648438,107.4010009765625 2.0139999389648438,107.4010009765625 C2.0139999389648438,107.4010009765625 -38.262001037597656,107.4010009765625 -38.262001037597656,107.4010009765625 C-38.262001037597656,107.4010009765625 -38.262001037597656,-107.4010009765625 -38.262001037597656,-107.4010009765625 C-38.262001037597656,-107.4010009765625 38.262001037597656,-107.4010009765625 38.262001037597656,-107.4010009765625 C38.262001037597656,-107.4010009765625 38.262001037597656,-67.1259994506836 38.262001037597656,-67.1259994506836 C38.262001037597656,-67.1259994506836 2.0139999389648438,-67.1259994506836 2.0139999389648438,-67.1259994506836z"
                            fill-opacity="0"
                            stroke="rgb(255,254,255)"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                            stroke-opacity="1"
                            stroke-width="2"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                      <g
                        style="display: none;"
                      >
                        <g>
                          <path />
                          <path
                            fill-opacity="0"
                            stroke-linecap="butt"
                            stroke-linejoin="miter"
                            stroke-miterlimit="10"
                          />
                        </g>
                      </g>
                    </g>
                  </svg>
                </div>
              </div>
              <span
                class="sr-only"
              >
                Feastings
              </span>
            </h1>
            <div
              class="text-content"
            >
              <h4>
                I am a pullquote
              </h4>
              <p>
                Dolore aliquip nostrud culpa ut nisi do do magna elit. Proident quis nostrud adipisicing ea aliqua incididunt. Lorem occaecat quis esse velit exercitation fugiat ipsum reprehenderit nulla ex ut deserunt. Officia dolore eiusmod et nulla aliquip ea ad cillum.
              </p>
            </div>
          </div>
          <div
            class="imageFrame__Container-sc-180q2f3-0 cA-DGOH imageframe__container desktop-image-frame"
          >
            <div
              class="imageframe__image-container"
            >
              <div
                class="imageComponent__StyledImage-sc-y6629v-0 iprdOX image-component__container"
                type=""
              >
                <div
                  style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                >
                  <img
                    alt="Lady reclining under giant glitter ball in the style of a christmas pudding"
                    class="imageframe__image"
                    data-nimg="fill"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; filter: blur(20px); background-size: cover; background-image: url(/img/homePage/xmas-glitter-ball-reclining-lady-rct-2.webp); background-position: 0% 0%;"
                  />
                  <noscript />
                </div>
              </div>
            </div>
            <div
              class="frame__Container-sc-he37x9-0 gFwSUZ svg-frame-container svg__frame--mobile"
            >
              <svg
                class="svg__frame--mobile"
                viewBox="0 0 1180 1450"
                xml:space="preserve"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="frame__corner"
                  d="M0 0v1450h1180V0H0zm1113.7 1222.6c0 9.8-7.3 18.1-17.1 19.6-62.8 9.5-112.5 59.9-121.9 123.4-1.4 9.9-9.8 17.2-19.7 17.2H225c-9.9 0-18.2-7.4-19.7-17.2-9.3-63.5-59.1-113.8-121.9-123.4-9.8-1.5-17.1-9.7-17.1-19.6V225.4c0-9.8 7.3-18.1 17.1-19.6 62.8-9.5 112.5-59.9 121.9-123.4 1.4-9.9 9.8-17.2 19.7-17.2h730c10 0 18.3 7.3 19.7 17.2 9.3 63.5 59.1 113.8 121.9 123.4 9.8 1.5 17.1 9.7 17.1 19.6v997.2z"
                />
                <path
                  class="frame__in"
                  d="M966.3 50H213.7c-10.2 0-18.8 7.6-20.3 17.7-9.7 65.4-61 117.4-125.7 127.3-10.2 1.5-17.7 10-17.7 20.2v1019.6c0 10.2 7.5 18.7 17.7 20.2 64.7 9.9 116 61.8 125.7 127.3 1.5 10.1 10.1 17.7 20.3 17.7h752.7c10.2 0 18.8-7.6 20.3-17.7 9.7-65.5 61-117.4 125.7-127.3 10.1-1.5 17.7-10.1 17.7-20.2V215.2c0-10.2-7.5-18.7-17.7-20.2-64.7-9.9-116-61.8-125.7-127.3-1.5-10.1-10.1-17.7-20.4-17.7z"
                />
                <path
                  class="frame__out"
                  d="M955 66.2H225c-9.9 0-18.3 7.3-19.7 17.2-9.4 63.5-59.1 113.9-121.9 123.4-9.8 1.5-17.1 9.8-17.1 19.6v997.1c0 9.9 7.3 18.1 17.1 19.6 62.8 9.6 112.6 59.9 121.9 123.4 1.5 9.8 9.8 17.2 19.7 17.2h730c9.9 0 18.3-7.3 19.7-17.2 9.4-63.5 59.1-113.9 121.9-123.4 9.8-1.5 17.1-9.8 17.1-19.6V226.4c0-9.9-7.3-18.1-17.1-19.6-62.8-9.6-112.6-59.9-121.9-123.4-1.4-9.9-9.7-17.2-19.7-17.2z"
                />
              </svg>
            </div>
            <div
              class="frame__Container-sc-he37x9-0 dXCvsx svg-frame-container svg__frame--desktop"
            >
              <svg
                class="svg__frame--desktop"
                viewBox="0 0 2020 1580"
                xml:space="preserve"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="frame__corner"
                  d="M0,0v1580h2020V0H0z M1953.7,1155c0,9.9-7.4,18.2-17.2,19.7c-63.5,9.3-113.8,59.1-123.4,121.9
    c-1.5,9.8-9.7,17.1-19.6,17.1H226.4c-9.8,0-18.1-7.3-19.6-17.1c-9.5-62.8-59.9-112.5-123.4-121.9c-9.9-1.4-17.2-9.8-17.2-19.7V425
    c0-10,7.3-18.3,17.2-19.7c63.5-9.3,113.8-59.1,123.4-121.9c1.5-9.8,9.7-17.1,19.6-17.1h1567.1c9.8,0,18.1,7.3,19.6,17.1
    c9.5,62.8,59.9,112.5,123.4,121.9c9.9,1.4,17.2,9.8,17.2,19.7V1155z"
                />
                <path
                  class="frame__in"
                  d="M50,413.7v752.7c0,10.2,7.6,18.8,17.7,20.3c65.4,9.7,117.4,61,127.3,125.7c1.5,10.1,10.1,17.7,20.2,17.7
		h1589.6c10.2,0,18.7-7.5,20.2-17.7c9.9-64.7,61.8-116,127.3-125.7c10.1-1.5,17.7-10.1,17.7-20.3V413.7c0-10.2-7.6-18.8-17.7-20.3
		c-65.5-9.7-117.4-61-127.3-125.7c-1.5-10.2-10-17.7-20.2-17.7H215.2c-10.2,0-18.7,7.5-20.2,17.7c-9.9,64.7-61.8,116-127.3,125.7
		C57.6,394.8,50,403.4,50,413.7z"
                />
                <path
                  class="frame__out"
                  d="M66.2,425v730c0,9.9,7.3,18.3,17.2,19.7c63.5,9.4,113.9,59.1,123.4,121.9c1.5,9.8,9.8,17.1,19.6,17.1h1567.1
		c9.9,0,18.1-7.3,19.6-17.1c9.6-62.8,59.9-112.6,123.4-121.9c9.8-1.5,17.2-9.8,17.2-19.7V425c0-9.9-7.3-18.3-17.2-19.7
		c-63.5-9.4-113.9-59.1-123.4-121.9c-1.5-9.8-9.8-17.1-19.6-17.1H226.4c-9.9,0-18.1,7.3-19.6,17.1c-9.6,62.8-59.9,112.6-123.4,121.9
		C73.5,406.7,66.2,415,66.2,425z"
                />
              </svg>
            </div>
          </div>
        </section>
      </div>
      <div
        class="ThemeContext__ColoredSurface-sc-7o1m9v-0 rMDNb colored-surface"
      >
        <div
          class="floatingNav__Wrapper-sc-16mop7q-0 gecmAm"
          data-test-id="floatingNav"
          draggable="false"
          style="user-select: none;"
        >
          <button
            class="circleButton__Wrapper-sc-o294s7-0 kmASHX"
          >
            <svg
              fill="currentColor"
              height="25"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="25"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M20.5 6c-2.61.7-5.67 1-8.5 1s-5.89-.3-8.5-1L3 8c1.86.5 4 .83 6 1v13h2v-6h2v6h2V9c2-.17 4.14-.5 6-1l-.5-2zM12 6c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
              />
            </svg>
          </button>
          <button
            class="circleButton__Wrapper-sc-o294s7-0 kmASHX"
          >
            <svg
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"
              />
              <polyline
                points="16 6 12 2 8 6"
              />
              <line
                x1="12"
                x2="12"
                y1="2"
                y2="15"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
